common_scripts/requirements.txt

```diff
@@ -4,4 +4,8 @@ azure-storage-blob==12.0.0b4
 requests
 ecdsa
 pycryptodome
-web3
\ No newline at end of file
+typing_extensions
+web3
+flask
+flask_cors
+flask_restplus
```

Comment:
We should run a pip freeze and add specific versions to this to avoid library upgrades breaking our code

Reply:
Good point, fixed in 7e0ae9a